src/axom/quest/tests/quest_regression.cpp

```diff
@@ -408,19 +408,10 @@ void runContainmentQueries(Input& clargs)
  */
 void runDistanceQueries(Input& clargs)
 {
-  constexpr int maxDepth = 10;
-  constexpr int maxEltsPerBucket = 25;
-
-  SLIC_INFO(
-    fmt::format("Initializing BVH tree (maxDepth: {}, "
-                "maxEltsPerBucket: {}) over mesh '{}'...",
-                maxDepth,
-                maxEltsPerBucket,
-                clargs.meshName));
+  SLIC_INFO(fmt::format("Initializing linear BVH tree over mesh '{}'...",
```

Comment:
Given our conversation on monday, we should be consistent in our messaging/documentation about "BVH" over "BVH tree"